pyproject.toml

```
@@ -17,7 +17,7 @@ dependencies = [
     "six>=1.10",
     # Not directly used on the client, but some server-side environments have
     # tblib transitively enabled, so this is needed for unpickling.
-    "tblib~=1.7",
+    "tblib",
```

Based on the timeline, I think we need to treat `1.7` as a lower bound.
```suggestion
    "tblib>=1.7",
```